### PR TITLE
CA-781 Deafening reporter's tab when zoiper call is present

### PR DESCRIPTION
**Pull request by Galzk**
Do not merge without tests!

### CODE REVIEW

**Comment by Galzk** [comment]
Demo:
https://user-images.githubusercontent.com/34302982/113740894-ce811d00-9709-11eb-934e-50ad52111898.mp4

File: src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
 import EventEmitter from 'events';
 import React from 'react';
-import ParticipantStripGrid, { ParticipantStripGridProps } from './ParticipantStripGrid';
+import ParticipantStripGrid from './ParticipantStripGrid';
 import { shallow } from 'enzyme';
 import useSelectedParticipant from '../../VideoProvider/useSelectedParticipant/useSelectedParticipant';
 import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
+import { useAppState } from '../../../state';
```
Comment (talr) [inline src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx]: Fix per our convention,

File: src/hooks/useIsDeafend/useIsDeafened.test.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import EventEmitter from 'events';
+import useIsDeafened from './useIsDeafened';
+import useVideoContext from '../useVideoContext/useVideoContext';
+import { PARTICIPANT_TYPES } from '../../utils/rbac/ParticipantTypes';
+import { useAppState } from '../../state';
```
Comment (talr) [inline src/hooks/useIsDeafend/useIsDeafened.test.ts]: Same

**Review by talr — CHANGES_REQUESTED** [review]

**Comment by Galzk** [comment]
I'm unable to check if the mute functionality is working because of an issue on my local env, leaving this for now

File: src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx (hunk 1 above)
Comment (Galzk) [reply src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx]: https://github.com/eScribers/twilio-video-app-react/pull/72
Merge it to solve this conversation

File: src/hooks/useIsSilenced/useIsSilenced.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  const {
+    room: { localParticipant },
+  } = useVideoContext();
+  const { isSilenced, setIsSilenced, setNotification } = useAppState();
+  const participants = useParticipants();
+  const participantCommands = useParticipant();
+
+  useEffect(() => {
+    const isReporter = ParticipantIdentity.Parse(localParticipant.identity)['partyType'] === PARTICIPANT_TYPES.REPORTER;
+    const isZoiperConnected =
```
Comment (talr) [inline src/hooks/useIsSilenced/useIsSilenced.tsx]: isSipClientConnected

**Review by talr — APPROVED** [review]

**Pull request merged**